app/main.jsx

```diff
@@ -53,16 +53,16 @@ const fetchInitialData = (nextRouterState) => {
 }
 
 const onAppEnter = () => {
-  Promise.all([
-    axios.get('/api/products'),
-    axios.get('/api/reviews'),
-  ])
-  .then(responses => responses.map(r => r.data))
-  .then(([products, reviews]) => {
-    store.dispatch(setProducts(products))
-    store.dispatch(setReviews(reviews))
-  })
-  .catch(console.error)
+  // Promise.all([
```

Comment:
commented all of this out because it will keep failing in the console which is annoying! will make a separate issue to fix why this is erroring